.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: python
+
+python:
+  - 2.7
+  - 3.3
+  - 3.4
+
+env:
+  - DEPS="numpy=1.8 matplotlib=1.3 jinja2=2.7.2 pandas=0.13.1 nose"
+
+install:
+  - conda create -n testenv --yes python=$TRAVIS_PYTHON_VERSION
+  - source activate testenv
+  - conda install --yes $DEPS
+  - python setup.py install
+
+before_install:
+  # setup virtual x
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  # then install python version to test
+  - if [ ${TRAVIS_PYTHON_VERSION:0:1} == "2" ]; then wget http://repo.continuum.io/miniconda/Miniconda-2.2.2-Linux-x86_64.sh -O miniconda.sh; else wget http://repo.continuum.io/miniconda/Miniconda3-2.2.2-Linux-x86_64.sh -O miniconda.sh; fi
```

Comment:
@damianavila : Thanks, that sounds like a good idea.  I've tried to break this out into a separate PR and make the change you propose in #41.  Can you let me know there if this is what you meant?